docs/scenarios.md

```diff
@@ -0,0 +1,53 @@
+# Use Cases
+
+This doc provides sample Ark commands for the following common scenarios:
+* [Disaster recovery][0]
+* [Cluster migration][1]
+
+## Disaster recovery
+
+*Using Schedules and Restore-Only Mode*
+
+If you periodically back up your cluster's resources, you are able to return to a previous state in case of some unexpected mishap, such as a service outage. Doing so with Heptio Ark looks like the following:
+
+1. After you first run the Ark server on your cluster, set up a daily backup (replacing `<SCHEDULE NAME>` in the command as desired):
+
+    ```
+    ark schedule create <SCHEDULE NAME> --schedule "0 7 * * *"
+    ```
+    This creates a Backup object with the name `<SCHEDULE NAME>-<TIMESTAMP>`.
+
+2. A disaster happens and you need to recreate your resources.
+
+3. Update the [Ark server Config][3], setting `restoreOnlyMode` to `true`. This prevents Backup objects from being created or deleted during your Restore process.
+
+4. Create a restore with your most recent Ark Backup:
+    ```
+    ark restore create <SCHEDULE NAME>-<TIMESTAMP>
```

Comment:
@skriss fyi this will need to be changed after #342 goes in

Reply:
@Bradamant3 #342 has merged, so this needs to be `ark restore create --from-backup <SCHEDULE NAME>-<TIMESTAMP>`